workorders.lic

```diff
@@ -623,11 +630,11 @@ class WorkOrders
         stow_hands
         exit
       else
+		volume = deed_ingot_volume
         fput('tap deed')
         pause
         fput("get #{@use_own_ingot_type} ingot") unless checkleft || checkright
       end
-      volume = ingot_volume
```

Comment:
Do you need to do another `unless volume` check here?

Reply:
this spot will only get here if the item is a deed, so its really not needed at all